src/Nexcess/GithubWebhookJira/Webhook.php

```diff
@@ -96,6 +102,30 @@ public function __construct(Application $app, Request $request) {
     $this->_secret = getenv('SECRET');
     $this->_issue_prefix = getenv('JIRA_ISSUE_PREFIX');
     $this->_jira_url = getenv('JIRA_URL');
+    $this->_transition_opened = (int) getenv('JIRA_TRANSITION_OPENED');
```

Comment:
grabbing the env from the web index.php file seems even more wrong. The index file is just a bootstrap, it shouldnt care about your configuration. I could move it into a `Config` class and instantiate it in the constructor of the Webhook class maybe.

Reply:
> The index file is just a bootstrap, it shouldnt care about your configuration

the bootstrap is where we create the application, and part of that process is to make sure it has the dependencies it needs (such as configuration settings).  this would give us control over them (for example, we might want to be able to pick from multiple credentials based on user selection in the future).  It would also make testing easier.

for our immediate purposes, i realize this is not an issue.  I'll leave it to your judgement whether it should be done now or when/if needed.